rs/sns/governance/src/governance.rs

```diff
@@ -2545,6 +2546,33 @@ impl Governance {
             })
     }
 
+    pub fn check_no_upgrades_in_progress(&self) -> Result<(), GovernanceError> {
```

Comment:
Let's add a comment to explain why we need two similar-looking functions to co-exist.

Reply:
Addressed here https://github.com/dfinity/ic/pull/2034#discussion_r1831410370